plot_app/configured_plots.py

```diff
@@ -269,9 +269,9 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data):
     data_plot.change_dataset('position_setpoint_triplet')
     data_plot.add_circle(['current.alt'], [plot_config['mission_setpoint_color']],
                          ['Altitude Setpoint'])
-    data_plot.change_dataset('actuator_controls_0')
```

Comment:
Please do not remove existing data w/o good reason. It's generally useful for MC's.
To improve the plot regarding the offset difference, you could add the minimum baro altitude (rounded down to the next lower 100) as an offset to the thrust. Then everything is in a similar range.